src/SDL12_compat.c

```diff
@@ -1018,6 +1018,8 @@ static SDL12_EventFilter EventFilter12 = NULL;
 static SDL12_Cursor *CurrentCursor12 = NULL;
 static Uint8 EventStates[SDL12_NUMEVENTS];
 static int SwapInterval = 0;
+static float OpenGLBuffersSwapTickInteval = 0.f;
```

Comment:
typo
```suggestion
static float OpenGLBuffersSwapTickInterval = 0.f;
```

(also fix its uses)